src/lib/viewers/BaseViewer.js

```diff
@@ -215,7 +215,7 @@ class BaseViewer extends EventEmitter {
         }
 
         if (this.options.enableAnnotationsDiscoverability) {
-            this.rootEl.classList.add('bp-annotations-discoverable');
+            this.containerEl.classList.add('bp-annotations-discoverable');
```

Comment:
Should we also modify the stylesheet?

Reply:
Yep I've pushed a fix to nest the styles under `bp-content`.